dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/sqlclassification/KyuubiGetSqlClassification.scala

```diff
@@ -35,28 +35,25 @@ import org.apache.kyuubi.sql.KyuubiSQLConf._
  *  If there have no this named jsonFile,
  *  the service will upload the default matching rule: sql-classification-default.json.
  */
-object KyuubiGetSqlClassification {
-
+object KyuubiGetSqlClassification extends Logging {
   private val jsonNode: Option[JsonNode] = {
     SQLConf.get.getConf(SQL_CLASSIFICATION_ENABLED) match {
       case true =>
         val objectMapper = new ObjectMapper
-        var url: URL = Thread.currentThread().getContextClassLoader
-          .getResource("sql-classification.json")
+        var url: URL = getClass.getClassLoader.getResource("sql-classification.json")
```

Comment:
why you use getClass.getClassLoader to get resource?

Reply:
It's better to keep these consistent, user can still specify the custom config.